rules-tests/Arguments/Rector/FuncCall/FunctionArgumentDefaultValueReplacerRector/Fixture/version_compare.php.inc

```diff
@@ -6,6 +6,7 @@ class VersionCompare
 {
     public function run()
     {
+        version_compare($a, $b, $c);
```

Comment:
Add a new case where previously we would have reported the rule as applied even though our rule did not change the code